@oschuett 
2024.1 does not work with BUILD_DBCSR=ON anymore
cmake results in the following error:
```
CMake Error at CMakeLists.txt:594 (find_package):
  By not providing "FindDBCSR.cmake" in CMAKE_MODULE_PATH this project has
  asked CMake to find a package configuration file provided by "DBCSR", but
  CMake did not find one.

  Could not find a package configuration file provided by "DBCSR" (requested
  version 2.6) with any of the following names:

    DBCSRConfig.cmake
    dbcsr-config.cmake

  Add the installation prefix of "DBCSR" to CMAKE_PREFIX_PATH or set
  "DBCSR_DIR" to a directory containing one of the above files.  If "DBCSR"
  provides a separate development package or SDK, be sure it has been
  installed.

```
https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=852134&view=logs&j=9e75aa61-5b36-561b-5f10-ea64a909c89f&t=6f1dc2fb-a6cb-5c46-0a92-93ce551ee674
Any pointers? Do we need another dbcsr release, too, maybe?

--- a/recipe/build.sh
+++ b/recipe/build.sh
@@ -26,10 +26,30 @@ else
  CP2K_USE_MPI=ON
  CP2K_VERSION=psmp
 fi
+
+export PKG_CONFIG_PATH=$PREFIX/lib:$PKG_CONFIG_PATH
+
+mkdir -p build_dbcsr
+pushd build_dbcsr
+
+cmake -DCMAKE_INSTALL_PREFIX=${PREFIX} \
+      -DCMAKE_BUILD_TYPE=Release \
+      -DCMAKE_INSTALL_LIBDIR=lib \
+      -DCMAKE_FIND_FRAMEWORK=NEVER \
+      -DCMAKE_FIND_APPBUNDLE=NEVER \
+      -DUSE_MPI=$CP2K_USE_MPI \
+      -DUSE_SMM=libxsmm \
+      ${CMAKE_ARGS} \
+      ../exts/dbcsr
+
+cmake --build . --config Release -j 2
+make install
+
+popd #build_dbcsr
+
 mkdir -p build
 pushd build
 
-export PKG_CONFIG_PATH=$PREFIX/lib:$PKG_CONFIG_PATH
 cmake -DCMAKE_INSTALL_PREFIX=${PREFIX} \
       -DCMAKE_BUILD_TYPE=Release \
       -DCMAKE_INSTALL_LIBDIR=lib \
@@ -37,7 +57,7 @@ cmake -DCMAKE_INSTALL_PREFIX=${PREFIX} \
       -DCMAKE_FIND_APPBUNDLE=NEVER \
       -DCP2K_ENABLE_REGTESTS=ON \
       -DPython3_EXECUTABLE="$PYTHON" \
-      -DCP2K_BUILD_DBCSR=ON \
+      -DCP2K_BUILD_DBCSR=OFF \
       -DCP2K_USE_MPI=$CP2K_USE_MPI \
       -DCP2K_BLAS_VENDOR=$BLAS_VENDOR \
       -DCP2K_SCALAPACK_VENDOR=$SCALAPACK_VENDOR \
@@ -58,7 +78,7 @@ export CP2K_DATA_DIR=$PWD/data
 
 if [ "$CP2K_USE_MPI" == "ON" ]
 then
-    python ./tools/regtesting/do_regtest.py cmake_build_cpu ${CP2K_VERSION} --smoketest --mpiexec 'mpiexec --bind-to none -mca plm isolated' --ompthreads 1
+    python ./tests/do_regtest.py local ${CP2K_VERSION} --smoketest --mpiexec 'mpiexec --bind-to none -mca plm isolated' --ompthreads 1
 else
-    python ./tools/regtesting/do_regtest.py cmake_build_cpu ${CP2K_VERSION} --smoketest --ompthreads 2
+    python ./tests/do_regtest.py local ${CP2K_VERSION} --smoketest --ompthreads 2
 fi

--- a/recipe/meta.yaml
+++ b/recipe/meta.yaml
@@ -1,5 +1,5 @@
 {% set name = "cp2k" %}
-{% set version = "2023.2" %}
+{% set version = "2024.1" %}
 
 # Define build matrix for MPI vs. non-mpi
 # ensure mpi is defined (needed for conda-smithy recipe-lint)
@@ -17,7 +17,7 @@ package:
 
 source:
   - url: https://github.com/{{ name }}/{{ name }}/releases/download/v{{ version }}/{{ name }}-{{ version }}.tar.bz2
-    sha256: adbcc903c1a78cba98f49fe6905a62b49f12e3dfd7cedea00616d1a5f50550db
+    sha256: a7abf149a278dfd5283dc592a2c4ae803b37d040df25d62a5e35af5c4557668f
     patches:
       - macosxtest.patch  # [osx]
   - url: https://github.com/cp2k/dbcsr/releases/download/v2.6.0/dbcsr-2.6.0.tar.gz
@@ -31,8 +31,8 @@ build:
   # - nompi: osx issue introduced in 8.1, fixed in future 8.2
   # - openmpi: runtime failures during test suite
   skip: true  # [not linux]
-  # For version 2023.2 the tests fail with segmentation fault with MKL and OpenMPI
-  # in line 452 of cp_basic_linalg.f, i.e.
+  # For version 2024.1 the tests fail with segmentation fault with MKL and OpenMPI
+  # in line 455 of cp_basic_linalg.f, i.e.
   #          CALL pdgemm(transa, transb, m, n, k, alpha, a, i_a, j_a, desca, b, i_b, j_b, &
   #                   descb, beta, c, i_c, j_c, descc)
   # (specific to scalapack MKL and tested as failing for MKL 2019 until 2024

